scrapers/Baberotica.yml

```diff
@@ -36,8 +36,8 @@ xPathScrapers:
           selector: //meta[@itemprop="url"]/@content
           postProcess:
             - map:
-                https://avidolz.com: AvIdolz
-                https://baberotica.com: Baberotica
-                https://baberoticavr.com: BaberoticaVR
-                https://teenthais.com: TeenThais
-# Last Updated June 30, 2021
+                https://avidolz.com/: AvIdolz
+                https://baberotica.com/: Baberotica
+                https://baberoticavr.com/: BaberoticaVR
+                https://teenthais.com/: TeenThais
+# Last Updated October 3, 2023
```

Comment:
This has the same issue as your other PR, the date needs to be "October 03, 2023" 😅